src/server/helpers/props.ts

```diff
@@ -16,8 +16,12 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+import {Request, Response} from 'express';
```

Comment:
Should this be `import type {Request, Response}…` ? 
Does it make any difference after transpilation?

Reply:
Should make no difference - Babel strips out all types and if there are no uses of imported symbols, it should remove the dependency too in the compiled code. I did a quick check of `lib/server/helpers/props.js` and looks like this holds true there.

babel-typescript has a [`onlyRemoveTypeImports` option](https://babeljs.io/docs/en/babel-preset-typescript#onlyremovetypeimports), which is false by default, implying that the default behavior is to remove all unused imports after transpilation.